modules/wordads/wordads.php

```diff
@@ -190,6 +190,10 @@ function insert_head_iponweb() {
 	 * @since 4.5.0
 	 */
 	function insert_ad( $content ) {
+		// Ad JS won't work in XML feeds.
+		if ( is_feed() ){
```

Comment:
nit-picking, but could you add a space after the closing parenthesis to make phpcs happy?

Thanks!